src/components/configMaps/ConfigMap.tsx

```diff
@@ -607,6 +609,26 @@ export function ConfigMapForm({
         }
         toggleYamlMode(not)
     }
+
+    const closeDeleteModal = (): void => {
+        setShowDeleteModal(false)
+    }
+
+    const setShowDelete = (): void => {
```

Comment:
Rename this to `onClickShowDelete` or something.

Reply:
Done